mesonbuild/backend/vs2010backend.py

```diff
@@ -946,7 +946,8 @@ def gen_vcxproj(self, target, ofname, guid):
                 self.add_project_reference(root, tvcxproj, tid)
             else:
                 # Other libraries go into AdditionalDependencies
-                additional_links.append(linkname)
+                if not linkname in additional_links:
```

Comment:
__[Flake8]__

[E713] test for membership should be 'not in'


<!-- Comment by [SideCI](https://sideci.com) -->